papis/base.py

```diff
@@ -76,7 +76,7 @@ def basedoc_to_papisdoc(basedoc: Dict[str, Any]) -> Dict[str, Any]:
         _key_translate("dcsubject", "tags", "m", lambda x: " ".join(x)),
         _key_translate("dcdoi", "doi", "m", lambda x: x[0]),
         _key_translate("dctype", "type", "m", lambda x: x[0].lower()),
-        _key_translate("dclang", "lang", "mutli", lambda x: x[0]),
+        _key_translate("dclang", "lang", "m", lambda x: x[0]),
```

Comment:
Is this correct? Not sure what that `mutli` was and the code didn't seem to check for it..

(context: `codespell` drew my attention by wanting to spell it as `multi`)

Reply:
That is probably correct yes, thanks!